web/src/dropdown_widget.ts

```diff
@@ -90,7 +90,7 @@ export class DropdownWidget {
     instance: tippy.Instance | undefined;
     default_id: string | number | undefined;
     current_value: string | number | undefined;
-    unique_id_type: DataTypes;
+    unique_id_type: DataTypes | undefined;
```

Comment:
I'm skeptical of these changes to the `DropdownWidget` type; can you explain why it's correct?

Reply:
In the `initialize()` function in `compose_recipient`, a new `DropDownWidget ` is initialized, however, the `unique_id_type` property wasn't used. The `setup` function from is called on this object. However, after inspecting the `setup` function, it isn't a must the `unique_id_type` is provided on the `DropDownWidget` object for the function to execute without any errors.